lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -630,32 +630,42 @@ export class UserAgentApplication {
      *
      */
     acquireTokenSilent(userRequest: AuthenticationParameters): Promise<AuthResponse> {
+        this.logger.info("AcquireTokenSilent has been called");
+        this.logger.verbose(`AcquireTokenSilent params:\n${JSON.stringify(userRequest)}`);
         // validate the request
         const request = RequestUtils.validateRequest(userRequest, false, this.clientId, Constants.interactionTypeSilent);
         const apiEvent: ApiEvent = this.telemetryManager.createAndStartApiEvent(request.correlationId, API_EVENT_IDENTIFIER.AcquireTokenSilent, this.logger);
         const requestSignature = RequestUtils.createRequestSignature(request);
 
         return new Promise<AuthResponse>((resolve, reject) => {
 
-            // block the request if made from the hidden iframe
+            // block the request if made from the hidden iframe TODO: Ask about why to extend comment
             WindowUtils.blockReloadInHiddenIframes();
 
             const scope = request.scopes.join(" ").toLowerCase();
+            this.logger.verbose(`Serialized scopes: ${scope}`);
 
             // if the developer passes an account, give that account the priority
             const account: Account = request.account || this.getAccount();
+            this.logger.verbosePii(`Set account to: ${JSON.stringify(account)}`);
 
-            // extract if there is an adalIdToken stashed in the cache
+            // extract if there is an adalIdToken stashed in the cache TODO: Ask about why
```

Comment:
This allows MSAL to SSO to an existing session that was started with ADAL and upgrade it to an MSAL session. This allows for seemless SSO when upgrading an app from ADAL -> MSAL.

Reply:
Updated comment to reflect the above.